repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -126,37 +126,27 @@ class ReposeJettyServer(val nodeContext: AbstractApplicationContext,
 
         //Handle the Protocols and Ciphers
         //varargs are annoying, so lets deal with this using the scala methods
+        // From https://www.eclipse.org/jetty/documentation/9.4.x/configuring-ssl.html
+        // "When working with Includes / Excludes, it is important to know that Excludes will always win."
         import scala.collection.JavaConversions._
-        Option(ssl.getExcludedProtocols).foreach(xp => cf.addExcludeProtocols(xp.getProtocol.toList: _*))
-        Option(ssl.getIncludedProtocols).foreach { ip =>
-          if (ip.getProtocol.nonEmpty) {
-            cf.setIncludeProtocols(ip.getProtocol.toList: _*)
-          }
-
-        }
-        Option(ssl.getExcludedCiphers).foreach { xc =>
-          cf.addExcludeCipherSuites(xc.getCipher.toList: _*)
-        }
-
-        Option(ssl.getIncludedCiphers).foreach { ic =>
-          if (ic.getCipher.nonEmpty) {
-            cf.setIncludeCipherSuites(ic.getCipher.toList: _*)
-          }
-        }
+        Option(ssl.getIncludedProtocols) foreach { ip => cf.setIncludeProtocols(ip.getProtocol.toList: _*) }
+        Option(ssl.getExcludedProtocols) foreach { ep => cf.setExcludeProtocols(ep.getProtocol.toList: _*) }
+        Option(ssl.getIncludedCiphers) foreach { ic => cf.setIncludeCipherSuites(ic.getCipher.toList: _*) }
+        Option(ssl.getExcludedCiphers).foreach { ec => cf.setExcludeCipherSuites(ec.getCipher.toList: _*) }
```

Comment:
Won't you still need to clear the default excluded ciphers to maintain the 9.2.0 behavior? It only comes into play when the user does not configure any excluded ciphers, but still.

Reply:
Also, I would probably just clear all of the includes/excludes for consistency.

Then in Repose 9.0 we can decide what we want the default includes/excludes to be.

Reply:
Updated.